src/components/curator-actions/fetch-signatories.ts

```diff
@@ -49,3 +80,53 @@ export async function fetchMultisigSignatories(curatorAddress: string): Promise<
 		return [];
 	}
 }
+
+/**
+ * Fetches multisig accounts that the signatory is part of.
+ *
+ * @param signatory signatory address.
+ * @returns array of multisig accounts that the signatory is part of, an empty array in case of error.
+ */
+export async function fetchMultisigAccount(signatory: string): Promise<MultisigInfo[]> {
+	const graphqlEndpoint = get(currentBlockchain).baseUrls.statescanGraphqlApi;
+	if (!graphqlEndpoint) {
+		return [];
+	}
+
+	try {
+		const query = `
+        query {
+			multisigAddresses(
+				signatory: "${signatory}" 
+				offset: 0
+				limit: 1000
+			) {
+				multisigAddresses {
+				threshold
+				signatories
+				address
+				}
+				total
+				offset
+			}
+        }`;
```

Comment:
```suggestion
		const query = `
      query {
			  multisigAddresses(
				  signatory: "${signatory}" 
				  offset: 0
				  limit: 1000
			  ) {
				  multisigAddresses {
				    threshold
				    signatories
				    address
				  }
			  }
      }`;
```
Would the query benefit from some formatting? Also the total and offset fields seem unnecessary.

Reply:
I got too used to spaces for indentation and used them here instead of tabs, don’t commit this suggestion as is